src/main/resources/com/rultor/agents/req/merge.sh

```diff
@@ -20,6 +20,12 @@ while [ $(git show-branch "${BRANCH}" 2>/dev/null | wc -l) -gt 0 ]; do
 done
 
 git checkout -B "${BRANCH}" "fork/${fork_branch}"
+git fetch origin
+if git log --name-status --oneline HEAD..origin/${head_branch} | grep ".rultor.yml"; then
+  echo "Since original repository has different version of .rultor.yml file you have to sync your fork with the upstream repository."
+  exit 1
+fi
```

Comment:
@xupyprmv @mkordas to me this check seems a little redundant to be honest, in fact it might even be a bit of a usability problem for users not used to consistently rebasing. But feel free to correct me if I'm wrong here :)

I mean we are not Travis. The whole point of this task is to run the merge result `rultor.yml`, not the one from master or the fork individually (**no individually is very important**). 
The `rultor.yml` executed must be exactly that, which will end up in the repo after the merge.
This is the only way to ensure that past issues with the merge result introducing some dependency issue from the interplay `rultor.yml` and source code will not happen ever again.
What you build is what you get so to say ;)